spec/pam_spec.rb

```diff
@@ -2,22 +2,11 @@
 
 describe 'rstudio::pam' do
   let(:chef_run) do
-    runner = ChefSpec::Runner.new platform: 'ubuntu', version: '12.04' do |node|
+    runner = ChefSpec::SoloRunner.new platform: 'ubuntu', version: '12.04' do |node|
       node.set['rstudio']['nginx']['server_name'] = 'test.example.com'
     end.converge(described_recipe)
   end
 
-  before(:each) do 
-    ChefSpec::Server.create_data_bag('users', {
-      'jstump' => {
-        :id => 'jstump',
-        :rstudio => {
-          :passwd => 'secret-stuff'
-        }
-      }
-    })
-  end
```

Comment:
Yeah so I updated this syntax to reflect the docs on data bags and chefspec here: https://github.com/sethvargo/chefspec#dsl

But I couldn't get rid of this error: `undefined method `create_data_bag' for nil:NilClass`.

Finally I removed the data bag create and got a pass. So I really don't know. Thoughts?